### ISSUE TITLE
Accessing PropTypes via the main React package is deprecated

### ISSUE THREAD

**Issue by pyramation** [SamyPesse/react-mathjax#5]
As the warning message says when you use this library: 

```
warning.js:36 Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```

**Comment by pyramation** [comment]
when this pull is sorted out (https://github.com/SamyPesse/react-mathjax/pull/8) this issue can be closed

**Comment by SamyPesse** [comment]
Fixed in `1.0.0`
